### DISCUSSION TITLE
process several files one time and export out result files independently, no need MBR or any comparison.

### DISCUSSION THREAD

**Discussion by Masonwei32** [vdemichev/DiaNN#933] [Q&A]
hello, there. I am newbe to learn data analysis by DIANN. I am wandering is there any possible to process several files one time and export out result files independently, no need MBR or any comparison. I am just willing to spare myself to do other thing without keeping eyes on my screen. Welcome to any comments and suggestions. I appreciate that!

**Comment by vdemichev** [comment]
Yes, 'unrelated runs' option in the GUI in combination with the the --individual-reports option (in 'Additional options').
